source/extensions/common/wasm/wasm.h

```diff
@@ -142,6 +143,7 @@ class PluginHandle : public PluginHandleBase {
                          std::static_pointer_cast<PluginBase>(plugin)),
         plugin_(plugin), wasm_handle_(wasm_handle) {}
 
+  Wasm* wasmOfHandle() { return wasm_handle_ != nullptr ? wasm_handle_->wasm().get() : nullptr; }
```

Comment:
PluginHandle already provides the `wasmHandle()` accessor, IMO this additional accessor feels redundant. Suggest removing it, and if calling code needs a raw `Wasm*`, the expression above can be used at the caller side.

Reply:
SGTM.